c10/core/impl/PyObjectSlot.cpp

```diff
@@ -35,11 +35,11 @@ PyObject* PyObjectSlot::_unchecked_untagged_pyobj() const {
 }
 
 PyInterpreter& PyObjectSlot::load_pyobj_interpreter() const {
-  auto interpreter = pyobj_interpreter_.load(std::memory_order_acquire);
-  if (interpreter) {
-    return *interpreter;
-  }
-  TORCH_CHECK(false, "cannot access PyObject for Tensor - no interpreter set");
+  auto* interpreter = getGlobalPyInterpreter();
```

Comment:
Not sure how this is different from pyobj_interpreter() above, but I guess we can delete as well? :)